classes/class-wc-connect-shipping-label.php

```diff
@@ -209,9 +175,17 @@ public function get_selected_packages( WC_Order $order ) {
 							$formatted = WC_Connect_Compatibility::instance()->get_formatted_variation( $product, true );
 							$product_data['attributes'] = $formatted;
 						}
+						$customs_info = get_post_meta( $product_data['product_id'], 'wc_connect_customs_info', true );
+						if ( $customs_info ) {
+							$product_data = array_merge( $product_data, $customs_info );
+						}
 					} else {
 						$product_data['name'] = WC_Connect_Compatibility::instance()->get_product_name_from_order( $product_data['product_id'], $order );
 					}
+					$product_data['value'] = WC_Connect_Compatibility::instance()->get_product_price_from_order( $product_data['product_id'], $order );
```

Comment:
We need to make sure to send the value even when `packages` is not defined. More details:

https://github.com/Automattic/woocommerce-services/issues/1465

Reply:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1382/commits/806e5820ba6c95ff24e2ff76b98e46e099f7c377 (I'm glad some bugs are actually easier than expected to fix)